corehq/apps/userreports/util.py

```diff
@@ -133,6 +135,7 @@ def number_of_ucr_reports(domain):
     return len(ucr_reports)
 
 
+@memoized
```

Comment:
This would be memoized at the module (and therefore process) level, right? I think that could be a memory problem for something long running like a pillow. Also, do you know how memoization works for a complex object like a `config`?

Reply:
(ignore)